minecraft_servers/craftergang/manifest.json

```diff
@@ -1,101 +1,97 @@
 {
-    "server_name": "craftergang",
-    "nice_name": "Craftergang",
-    "direct_ip": "craftergang.de",
-    "server_wildcards": [
-        "craftergang.net"
-    ],
-    "supported_languages": [
-        "de"
-    ],
-    "social": {
-        "discord": "https://discord.gg/ThCt8Pq",
-        "teamspeak": "craftergang.de",
-        "tiktok": "craftergang",
-        "youtube": "https://www.youtube.com/@craftergang",
-        "instagram": "craftergang"
+  "server_name": "craftergang",
+  "nice_name": "Craftergang",
+  "direct_ip": "craftergang.de",
+  "server_wildcards": [
+    "%.craftergang.net",
+    "%.craftergang.de"

```

Comment:
Not a real wildcard
<img width="837" alt="Shottr 2024-08-09 14 58 43" src="https://github.com/user-attachments/assets/e08c1062-2944-46eb-bd52-0b7e9482e551">